src/plugins/navigation/review/review.js

```diff
@@ -37,15 +37,15 @@ import navigatorFactory from 'taoQtiTest/runner/plugins/navigation/review/naviga
 var buttonData = {
     setFlag: {
         control: 'set-item-flag',
-        title: __('Flag the current item for later review'),
-        icon: 'anchor',
-        text: __('Flag for Review')
+        title: __('Bookmark the current question for later review'),
```

Comment:
I think wording and icon will change for 'normal' (not fizzy) layout too? Is that ok - shouldn't old layout stay the same?

Reply:
**nitpick:** Caution, this will impact every version of the component. It will introduce unwanted changes to the "standard" mode.

Reply:
I used the same flag here also